shared/wallets/confirm-form/index.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import Banner from '../banner'
+import type {Background} from '../../common-adapters/text'
+import Header from './header'
+import Participants from './participants'
+import NoteAndMemo from './note-and-memo'
+
+type ConfirmSendProps = {|
+  onClose: () => void,
+  onSendClick: () => void,
+  onBack: () => void,
+  amount: string,
+  assetType: string,
+  assetConversion?: string,
+  waiting?: boolean,
+  encryptedNote?: string,
+  publicMemo?: string,
+  bannerBackground?: Background,
+  bannerText?: string,
+|}
+
+const ConfirmSend = (props: ConfirmSendProps) => (
+  <Kb.MaybePopup onClose={props.onClose}>
+    <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true} style={styles.container}>
+      <Header
+        amount={props.amount}
+        assetType={props.assetType}
+        assetConversion={props.assetConversion}
+        onBack={props.onBack}
+      />
+      <Kb.ScrollView style={styles.scrollView}>
+        {!!props.bannerBackground &&
+          !!props.bannerText && <Banner background={props.bannerBackground} text={props.bannerText} />}
+        <Participants receivingUsername="nathunsmitty" receivingFullName="Nathan Smith" />
```

Comment:
PR looks good!  Except the `<Participants>` are hardcoded inside the component here.  Maybe that was intentional 'cause @zanderz has an open PR on the participants component?  Although his PR doesn't handle the to-another-wallet case.  So maybe we should agree on some props here?

Like:
```
toKeybaseUser: string
toStellarAddress: string
toAnotherWallet: string
```
and have the participants component check those in turn..?

Reply:
Yeah I wasn't entirely sure how I should handle the `Participants` component since I know @zanderz was working on it in #13168 but also the confirm screen's `Participants` were different enough to not totally match the send screen's. I think I would like to see the two components eventually be merged into one or at least share their common code, but that can probably happen when one (or both) of our PRs get merged in.

Definitely will add the extra props 👍